snapcraft/internal/states/_prime_state.py

```diff
@@ -41,18 +41,18 @@ def __init__(self, files, directories, dependency_paths=None,
             self.dependency_paths = dependency_paths
 
     def properties_of_interest(self, part_properties):
-        """Extract the properties concerning this step from part_properties.
+        '''Extract the properties concerning this step from part_properties.
 
         The only property of interest to the prime step is the `snap` keyword
         used to filter out files with a white or blacklist.
-        """
+        '''
```

Comment:
Yes I did! And he answered "three double quotes." Thus logic dictates that I replace all triple double quotes with single quotes, apparently.